daisy_workflows/image_build/enterprise_linux/kickstart/el8-post.cfg

```diff
@@ -1,10 +1,19 @@
 # Google Compute Engine kickstart config for Enterprise Linux 8.
-%post --log=/dev/ttyS0
-set -x
+%onerror
+echo "Build Failed!" > /dev/ttyS0
+shutdown -h now
+%end
 
+%post --erroronfail
+exec < /dev/tty3 > /dev/ttyS0
```

Comment:
still don't understand taking input from tty3
this section doesn't require input, and no input will likely be available from tty3?

Reply:
also you might want to redirect stderr to the console too

Reply:
Yeah- ok makes sense. I think the only use case is if you are using anaconda live or with interactive mode. We are not. Changed.